source/Sashimi.AzureCloudService/AzureCloudServiceServiceMessageHandler.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
+using System.Collections.Generic;
+using Octopus.Diagnostics;
+using Octostache;
+using Sashimi.AzureCloudService.Endpoints;
+using Sashimi.Server.Contracts.Accounts;
+using Sashimi.Server.Contracts.Endpoints;
+using Sashimi.Server.Contracts.ServiceMessages;
+
+namespace Sashimi.AzureCloudService
+{
+    class AzureCloudServiceServiceMessageHandler : ICreateTargetServiceMessageHandler
```

Comment:
I think the names of these create target handlers should have `CreateTarget` in them 

`AzureCloudServiceCreateTargetServiceMessageHandler `